examples/falcon_host_details.rs

```diff
@@ -51,7 +51,7 @@ async fn get_device_details(
     )
     .await?;
 
-    if response.errors.is_some() {
+    if !response.errors.is_empty() {
```

Comment:
I'm not sure about this change, isn't it the same?

Reply:
If you check [this change](https://github.com/CrowdStrike/rusty-falcon/pull/135/files#diff-c84b6d977020f0dafe666798ca03d44b562c2ebafa625ce899ff5a4d6350525cR16), it looks like `error` in no longer wrapped in an `Option` and is a "raw" `Vec` instead, unless I'm missing something?

Reply:
Makes sense, thanks for claryfication.